### ISSUE TITLE
How to train my own model with ms-issim? just change the lambda?  thanks for answer my question !!

### ISSUE THREAD

**Issue by Jerrry-Li** [InterDigitalInc/CompressAI#213]

**Comment by YodaEmbedding** [comment]
First train using MSE, then fine-tune with MS-SSIM.

https://interdigitalinc.github.io/CompressAI/zoo.html

> MS-SSIM optimized networks were fine-tuned from pre-trained MSE networks (with a learning rate of 1e-5 for both optimizers).

**Comment by Jerrry-Li** [comment]
> First train using MSE, then fine-tune with MS-SSIM.
> 
> https://interdigitalinc.github.io/CompressAI/zoo.html
> 
> > MS-SSIM optimized networks were fine-tuned from pre-trained MSE networks (with a learning rate of 1e-5 for both optimizers).

Thanks  you !!! But i still don't know how to use it. Can you show me your code? just for ms-ssim training .When I training my own model ,how to use the ms-ssim  metric

**Comment by fracape** [comment]
Yes, I merged it. Note that I haven't added the option in examples/codec.py but it's straightforward to add

**Comment by Jerrry-Li** [comment]
One more question is, where did you download the CLIC2020-PRO dataset you used and can you provide a link? I have read many articles where the BPG data on CLIC2020-PRO is different from the data you provided
